packages/fxa-auth-server/fxa-oauth-server/lib/grant.js

```diff
@@ -176,7 +176,7 @@ function generateIdToken(grant, access) {
     //iss set in jwt.sign
     iat: now,
     exp: now + ID_TOKEN_EXPIRATION,
-    at_hash: util.generateTokenHash(access.token),
+    at_hash: util.generateTokenHash(hex(access.token)),
```

Comment:
I think you'll want to use ` access.jwt_token || access.token.toString('hex')` here to be consistent with how `result.access_token` is calculated above. Alternately, we could pass in `result.access_token` as an explicit argument to this function, so that the "this is the final form of the token as seen by the client" logic is encapsulated in a single place.

Reply:
While we are at it, maybe we have an opportunity to make the interface consistent so we don't need to do `access.jwt_token`, maybe we can unify the interface just pass in `access.token` and have the right thing happen everywhere? No need to call `hex`, no need to do the `||`?

Reply:
Sold, updated 👍🏽